server/models/functions.sql

```diff
@@ -134,7 +134,7 @@ BEGIN
     SET accountType = 'expense';
   END IF;
 
-  SET accountTypeId = (SELECT id FROM account_type WHERE `type` = accountType COLLATE utf8_unicode_ci LIMIT 1);
+  SET accountTypeId = (SELECT id FROM account_type WHERE `type` = accountType COLLATE utf8mb4_unicode_ci LIMIT 1);
```

Comment:
Is it possible to get rid also of all these `COLLATE utf8mb4_unicode_ci` ?

Reply:
Hmm... good question.  Let me try!